packs/TTWDFL.json

```diff
@@ -0,0 +1,109 @@
+{
+    "cgdbId": 54,
+    "code": "TTWDFL",
+    "name": "The Things We Do For Love",
+    "releaseDate": null,
+    "cards": [
+        {
+            "code": "16011",
```

Comment:
Similarly, not sure how we want to do card codes. 16XXX is definitely the next cardspace in our current scheme.

I suspect each of these types of packs (if there are more) will have their own card numbering, e.g. the next one will start at card 1 rather than card 37. In such an event, that single pack would probably belong in a 17XXX cardspace. But until (if) that happens, it's hard to predict. 🤷‍♀

Reply:
not too concerned about it at this moment. 16XXX is correct for this one.